Runtime/BacktraceClient.cs

```diff
@@ -535,12 +536,13 @@ public void Refresh()
                 Database = GetComponent<BacktraceDatabase>();
                 if (Database != null)
                 {
+                    _breadcrumbs = Database.Breadcrumbs as BacktraceBreadcrumbs;
```

Comment:
we can't provide to our clients with a real implementation (we hide it behind the interface) of the BacktraceBreadcrumbs class. Because of that, we weren't able to log correctly system events when a user didn't add "manual" options in the breadcrumbs configuration (because we use flow that is dedicated to users and we check if user has enabled manual logs). Since we own this implementation and we always create BacktraceBreadcrumbs class instance casting here is safe. If you have other suggestions how we can handle it, please let me know. We also need to share reference between two classes.

Reply:
Small nit: if casting here is safe, why not direct cast it: `(BacktraceBreadcrubms)Database.Breadcrumbs`? That way, if something changes in the future (different implementation), it should fail here instantly and not silently.